content/doc/marketplace/_index.md

```diff
@@ -1,47 +1,30 @@
 ---
-type: docs
-title: Add-on API
-weight: 3
-shortdesc: How to integrate your own service as an add-on on Clever Cloud
-tags:
-- extend
+title: Marketplace APIs & Tools
+shortdesc: How to integrate your own service as an add-on on Clever Cloud's Marketplace
 keywords:
-- addon api
-- addon
+- Add-on
+- API
+- Marketplace
+- Partners
+aliases:
+- /doc/extend/addon-api/
 type: docs
----
-
-Clever Cloud offers you to sell your service as an add-on through the dashboard.
-This article will document how you can technically plug into the platform to provide add-on provision, deprovision, and configuration via the dashboard.
-
-There are two faces here:
-
-* [The add-on provider API](#add-on-provider-api) - The API *you* need to provide and document to allow Clever Cloud's backend to request provision and de-provision of add-ons.
-
-* [The add-on infos API](#add-on-infos-api) - The API Clever Cloud provides to allow you to get information about provisioned add-ons and their owners.
+comments: false
+weight: 13
 
-## Add-on Provider API
-
-This is the API *you* need to provide to allow Clever Cloud to provision an add-on for a customer.
-
-### How it works
-
-When a Clever Cloud customer uses the marketplace to provision an add-on, Clever Cloud sends a request to your service which provides a new add-on for this app.
+---
 
-### Build an add-on
+Clever Cloud allows its Marketplace partners to provide services as add-ons with revenue sharing. Thus, they can be available to purchase and provision from the [Console](https://console.clever-cloud.com), [Clever Tools](https://github.com/CleverCloud/clever-tools), the [API](/api) or other integrations such as the [Terraform provider](https://registry.terraform.io/providers/CleverCloud/clevercloud/).
```

Comment:
```suggestion
{< hextra/hero-subtitle >}} Clever Cloud allows its Marketplace partners to provide services as add-ons with revenue sharing. Thus, they can be available to purchase and provision from the [Console](https://console.clever-cloud.com), [Clever Tools](https://github.com/CleverCloud/clever-tools), the [API](/api) or other integrations such as the [Terraform provider](https://registry.terraform.io/providers/CleverCloud/clevercloud/).{{< /hextra/hero-subtitle >}}

```

Reply:
I prefer not to rely on hextra specific templating when not necessary